cmd/app/serve.go

```diff
@@ -195,6 +195,7 @@ func runServeCmd(cmd *cobra.Command, args []string) {
 	switch viper.GetString("ca") {
 	case "googleca":
 		baseca, err = googlecav1.NewCertAuthorityService(cmd.Context(), viper.GetString("gcp_private_ca_parent"))
+		defer baseca.Close()
```

Comment:
I believe I am not following you. What do you want before the switch ? Do you mean to add it here https://github.com/sigstore/fulcio/pull/930/files#diff-1627906dbce532aff8cc81f54790572e5ba04f3e568c88b6ed39a590642dafa4R141 ?

Reply:
sorry it wasn't clear! maybe something like:

```golang
var baseca certauth.CertificateAuthority
switch viper.GetString("ca") {
 ... // all of the switch statement stuff
}
defer baseca.Close()
```

so that Close() is called regardless of which CA we use.

Reply:
Done, thanks :).